scripts/build.js

```diff
@@ -4,6 +4,8 @@
 process.env.BABEL_ENV = 'production';
 process.env.NODE_ENV = 'production';
 
+console.log('Running ./scripts/build.js');
```

Comment:
this probably needs to be removed, or because it's in the build - it's fine to leave it there since it's not part of our run-time?

Reply:
Nope that can be removed. i forgot to remove that as test.